docs/content/sftp.md

```diff
@@ -279,6 +279,10 @@ option before first run, the auto-detection will be skipped, and if
 you set a different value later this will override any existing.
 Value `none` can be set to avoid any attempts at executing shell
 commands, e.g. if this is not allowed on the server.
+However, if you have “shell_type = none” in the configuration then you
```

Comment:
> “shell_type = none”

This uses the pesky "curly quotes". I wouldn't use those. And maybe then even make it a code formatted instead of quoted, like other references to `shell_type` and `none` in the lines above.

Reply:
yeap. will change